static/static/css/index-style.css

```diff
@@ -0,0 +1,59 @@
+body {
```

Comment:
Copied from Via 3. Although it would be super nice if this CSS had anything to do with the other CSS we have